src/prefect/server/database/migrations/versions/sqlite/2022_04_23_114831_fd966d4ad99c_rename_block_to_blockbasis_and_.py

```diff
@@ -15,10 +15,7 @@
 
 
 def upgrade():
-    op.rename_table("block_spec", "block_schema")
-    op.rename_table("block", "block_document")
-
-    with op.batch_alter_table("block_document", schema=None) as batch_op:
+    with op.batch_alter_table("block", schema=None) as batch_op:
```

Comment:
Renaming the table before doing batch alter does not work with the new transactional model. It fails complaining about a missing table.